### PR TITLE
Fix more broken links

### PR DESCRIPTION
**Pull request by amclark42**
Fixing more broken links found during static site generation. Most of the things fixed here were problems in the main DHQ site as well as my static copy.

- The left sidebar has updated links for "Announcements" **(let me know if these should be changed!)**:
  - [Call for Reviewers](https://digitalhumanities.org/dhq/news/news.html#peer_reviews)
  - [Call for Submissions](https://digitalhumanities.org/dhq/submissions/index.html#logistics)
- Custom article stylesheets now import the regular article XSLT, not the "editorial" one.
- Whitespace in the `<teiHeader>` entries for volume, issue, and article ID is now normalized in both the title and author indexes.
- Strange URLs with the regex pattern `\.{3,}000\d\d\d/resources/images` or `\.{2,}/000\d\d\d/resources/images` have been adjusted to just `resources/images`. (Most of these work in DHQ, thanks to Cocoon magic.)
- The `$session` parameter set in `template_preview_bios.xsl` has been commented out. This should keep a "Log out" button from appearing on `/preview/bios.html`.
- The indexes for titles and authors no longer place `@href` on `<a>` when an article is missing its volume and/or issue number in the `<teiHeader>`.

I also replaced the alt text on the image that forms the border of the left-hand sidebar. Since it's decorative, the `@alt` attribute has an empty string indicating that screen readers should ignore it.

### CODE REVIEW

**Comment by juliaflanders** [comment]
Ash, two comments:

1. For the Announcements links, I am not sure the update here will fix the real problem, which is that old announcements get moved to a separate file (announcements_archive.html). So those links will still be broken, even though the file path is now more correct.
2. For the title/author indexes: articles that don't have volume/issue information in their headers would only be either unpublished (i.e. in the internal preview area) or broken--either way, they should not be showing up in the author/title indexes. Could you say more about this scenario and what you found that made this change seem practically as well as theoretically useful?

**Comment by juliaflanders** [comment]
Addendum--we can pull in these changes and then @jawalsh  can separately go in and just remove the Announcements link in the left sidebar, which is obsolete.

**Comment by amclark42** [comment]
@juliaflanders Okay! I'll leave the Announcements removal to @jawalsh.

Some context for item 2: Articles without volumes or issues did in fact yield broken links. These articles had been published, but their `<teiHeader>`s were missing the necessary information to generate the links. Since Cocoon relies on the TOC to determine which articles go to which issue, this was only a problem on the indexes. I assumed that, since the articles have been published, they should still be listed in the index, even if they can't be linked to. (The function of the index is not just to _link_ but also to _list_, imo.)

Honestly, I would rather remove the `<a>` entirely when articles are erroneously published without a volume or issue number. However, because the indexes' stylesheets use a lot of XSLT 1.0 to do some complex stuff (complex for XSLT 1.0, anyway), I did not feel great about adding a lot of additional complexity to the code. I limited myself to only taking the `@href` off the `<a>`. When we upgrade these stylesheets to XSLT 2.0 or higher, we should definitely revisit what happens to articles with erroneous data.

**Tl;dr:** I felt it was more important to remove the programmatically-generated broken link now. But I also created [an issue signaling that we should come back to these stylesheets and upgrade them later](https://github.com/Digital-Humanities-Quarterly/dhq-journal/issues/69).

I'm sorry that I never let you know which articles were missing their volume/issue metadata, though. They're 000735 and 000664 — you can find them in the [Title index](http://digitalhumanities.org/dhq/index/title.html) by searching the page for `v n`.
